### DISCUSSION TITLE
tags pills position

### DISCUSSION THREAD

**Discussion by jromal** [squidfunk/mkdocs-material#3694] [Q&A] [answered]
Hi all.

I am managing the documentation in my department with mkdocs-material.
I very much like the tags function that has been released for all recently. 
But some of my colleagues like it less. They do not like the position of the "tag pill".

Is there a way over CSS or similar to "move" the pill to another place?
Maybe on the TOC side.

Any ideas or experiences with it?

Thanks.

**Comment by squidfunk** [comment]
You can [override the `content` partial](https://squidfunk.github.io/mkdocs-material/customization/#overriding-partials) to change the position:

https://github.com/squidfunk/mkdocs-material/blob/366a5ea3a470357a6d908fe9e1023b1bb4262d3f/src/partials/content.html#L34-L37

**Comment by jromal** [comment, marked as answer]
Thanks Martin. 

Not sure it could be done overriding blocks but I managed to put the tags on the TOC side using ```partial``` files.

For this instead of changing the ```content.html``` I did following:

- Create an empty ```partials/tags.html```.
- Copy the original content of ```partials/tags.html``` to ```partials/tags-toc.html``` and change it to have each tag on one line.
- Include in ```partials/toc.html```a calll to the "tags-toc" file before the TOC content is generated.
